templates/vue/README.md

```diff
@@ -46,20 +49,20 @@ Cypress is the framework we use for end-to-end (e2e) testing. For it to work, it
 
 ```json
 {
-    "env": {
-        "BASE_URL": "http://localhost:8888",
-        "USER_TAPESTRY_NAME": "testing-user",
-        "ADMIN_USERNAME": "admin_user",
-        "ADMIN_PASSWORD": "admin_pass",
-        "SUBSCRIBER_USERNAME": "subscriber_user",
-        "SUBSCRIBER_PASSWORD": "subscriber_pass"
-    }
+  "env": {
+    "ADMIN_USERNAME": "admin_name",
+    "ADMIN_PASSWORD": "admin_password",
+    "SUBSCRIBER_USERNAME": "subscriber_name",
+    "SUBSCRIBER_PASSWORD": "subscriber_password"
+  },
+  "baseUrl": "http://localhost:8000",
+  "retries": 1
 }
 ```
 
 Copy the file to `cypress.json` and change the variables to match your own settings.
 
-* `BASE_URL` is the url of your local site.
+* `baseUrl` is the url of your local site.
```

Comment:
I think this is the URL that Cypress uses or that the docker version uses (local site is on port 8888, I believe)